assets/airlines/lan.json

```diff
@@ -2,30 +2,31 @@
     "icao": "lan",
     "name": "LATAM Airlines Chile",
     "callsign": {
-        "name": "LAN Chile",
+        "name": "LAN",
```

Comment:
The callsign _is_ `Lan Chile`

Reply:
Every source says something different... German wikipedia: LAN, English wikipedia: LAN Chile, planespotters.net: LAN. But youre an ATC so you probably know best, thanks

Reply:
The problem is that the speech synthesiser would say that as "L - A - N Chile". While the callsign is technically `LAN Chile`, `Lan Chile` will get us the desired result